tools/pubsys/src/aws/ami/register.rs

```diff
@@ -74,17 +72,17 @@ async fn _register_image(
     }
 
     // Prepare parameters for AMI registration request
-    let root_bdm = BlockDeviceMapping {
-        device_name: Some(ROOT_DEVICE_NAME.to_string()),
-        ebs: Some(EbsBlockDevice {
-            delete_on_termination: Some(true),
-            snapshot_id: Some(root_snapshot.clone()),
-            volume_type: Some(VOLUME_TYPE.to_string()),
-            volume_size: ami_args.root_volume_size,
-            ..Default::default()
-        }),
-        ..Default::default()
-    };
+    let root_bdm = BlockDeviceMapping::builder()
+        .set_device_name(Some(ROOT_DEVICE_NAME.to_string()))
+        .set_ebs(Some(
+            EbsBlockDevice::builder()
+                .set_delete_on_termination(Some(true))
+                .set_snapshot_id(Some(root_snapshot.clone()))
+                .set_volume_type(Some(VolumeType::from(VOLUME_TYPE)))
+                .set_volume_size(ami_args.root_volume_size)
+                .build(),
+        ))
+        .build();
```

Comment:
I might look into what defaults the `build()` method adds to the struct compared to what Rusoto does in `Default::default()`... just to be sure nothing odd happens here.